generators/page-template.js

```diff
@@ -0,0 +1,53 @@
+const { writeFileSync } = require('fs');
+const { join } = require('path');
+const prompts = require('prompts');
+
+const getScriptTemplate = (templateName, baseFileName) => `<?php
+/**
+ * Template Name: ${templateName}
+ */
+
+$context = Timber\\Timber::context();
+$timber_post = new Timber\\Post();
+$context['post'] = $timber_post;
+
+// Render HTML templates.
+render_with_password_protection( $timber_post, '${baseFileName}.twig', $context );
+`;
+
+const twigTemplate = `{% extends "layouts/base.twig" %}
+
+{% block content %}
+	<div class="obj-width-limiter">
```

Comment:
It looks like the base layout already wraps this content with a div including the class `obj-width-limiter`:

![image](https://github.com/sparkbox/sparkpress-wordpress-starter/assets/565751/9f8e1852-3272-4c60-b350-26a23185ec9e)

Should we remove it from here?

Reply:
I opted to remove it from the wrapper div in `base.twig` since while most of the time, we'd want the width limiter, doing that at the page level would make it easier to override if a page needed to be full bleed. All of the existing templates effectively have `.obj-width-limiter` on the wrapping element within the `content` block, so the generators should match that precedent.

Reply:
Perfect, thanks!